src/renderer/screens/account/EmptyStateAccount.js

```diff
@@ -43,6 +43,7 @@ class EmptyStateAccount extends PureComponent<Props, *> {
     if (!mainAccount) return null;
 
     const hasTokens = Array.isArray(mainAccount.subAccounts);
+    const isTezos = mainAccount.currency.family === "tezos";
```

Comment:
i think logic should instead be `const hasTokens = mainAccount.subAccounts && mainAccount.subAccounts.length && mainAccount.subAccounts[0].type === "TokenAccount"`

we shouldn't if tezos in any part of our app, the logic that is getting fixed here is about tokens

Reply:
agreed